src/main/kotlin/com/example/Food_API/controller/FoodController.kt

```diff
@@ -7,18 +7,19 @@ import org.springframework.web.bind.annotation.*
 @RestController
 @RequestMapping("/api/food")
 class FoodController() {
-    @GetMapping(produces = ["application/json"])
-    fun getRecipes(): String? {
+
+    @GetMapping("/{foodId}")
```

Comment:
For the `@GetMapping()` you can still add the `produces`. It would look something like `@GetMapping("/{foodId}", produces = ["application/json"])`